tests/test_java.py

```diff
@@ -344,6 +344,23 @@ def test_sanity_j9(
         assert_collapsed(process_collapsed)
 
 
+@pytest.mark.parametrize("in_container", [True])
+@pytest.mark.parametrize("image_suffix", ["_zing"])
+def test_sanity_zing(
+    tmp_path: Path,
+    application_pid: int,
+    assert_collapsed: AssertInCollapsed,
+) -> None:
+    with make_java_profiler(
+        frequency=99,
+        storage_dir=str(tmp_path),
+        java_async_profiler_mode="cpu",
+    ) as profiler:
+        assert "Zing" in get_java_version(psutil.Process(application_pid), profiler._stop_event)
+        process_collapsed = snapshot_one_collapsed(profiler)
+        assert_collapsed(process_collapsed)
+
+
```

Comment:
you can just parametrize everything instead of creating a dup test func.. (`image_suffix`, `java_async_profiler_mode` and `java_version_look_string`)